parsec/core/gui/create_org_widget.py

```diff
@@ -55,17 +54,18 @@ class CreateOrgUserInfoWidget(QWidget, Ui_CreateOrgUserInfoWidget):
     def __init__(self):
         super().__init__()
         self.setupUi(self)
-        self.line_edit_user_email.textChanged.connect(self.check_infos)
+        self.line_edit_user_email.validity_changed.connect(self.check_infos)
+        self.line_edit_user_email.setValidator(validators.EmailValidator())
```

Comment:
`.set_validator` ? (as it seems to be used in `parsec/core/gui/claim_device_widget.py`)